### DISCUSSION TITLE
is there a possibilitiy to change the opening size of dialogs?

### DISCUSSION THREAD

**Discussion by olisnr** [StefanSchippers/xschem#253] [Q&A]
most time i want to insert some parts via insert, i feel, that its more comfortable if i first adjust the dialog size to much-bigger and adjust the dialog-lists to the same size.
is it possible to make this default without changing the holly xschem.tcl?

its the same at other dialogs like file-open or save-as

**Comment by StefanSchippers** [comment]
The following settings have been added in latest xschem (these can be set in xschemrc file):
All dimensions are in screen pixels.
```
#### initial size of the load_file_dialog (the file selector) dialog box
set file_dialog_default_geometry 700x680+100+40
#### load_file_dialog directory list (+ recent if shown) X width in pixels
set file_dialog_sp0 350
#### load_file_dialog recent components list X width in pixels
set file_dialog_sp1 100
#### load_file_dialog components list Y height in pixels
set file_dialog_v_sp0 420
```

See image below:

![file_selector](https://github.com/user-attachments/assets/b02b96f2-c81d-4a1f-882c-7ef2d80e537a)

**Comment by olisnr** [comment]
great! thanks!
